cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -20,41 +20,40 @@
     <property>
         <name>root.namespace</name>
         <value>cdap</value>
-        <description>Specifies the root namespace</description>
+        <description>Namespace for this CDAP instance</description>
     </property>
 
     <property>
         <name>zookeeper.quorum</name>
         <value>127.0.0.1:2181/${root.namespace}</value>
-        <description>Specifies the zookeeper quorum string</description>
+        <description>Zookeeper address host:port</description>
```

Comment:
I think we should leave this as zookeeper quorum string. Any reason why we are changing the description.